hibernate-core/src/main/java/org/hibernate/id/uuid/UuidVersion6Strategy.java

```diff
@@ -34,7 +34,7 @@
 public class UuidVersion6Strategy implements UUIDGenerationStrategy, UuidValueGenerator {
 	public static final UuidVersion6Strategy INSTANCE = new UuidVersion6Strategy();
 
-	private static class Holder {
+	public static class Holder {
```

Comment:
I just noticed this is in an API package... ?

Maybe those `Holder` classes should be annotated with `@Incubating`?

Reply:
Or @Internal?

Reply:
Yes that's what I meant :)